config/routes.rb

```diff
@@ -28,15 +28,8 @@
   get "manifest" => "rails/pwa#manifest", :as => :pwa_manifest
 
   # Defines the root path route ("/")
-  root "pages#inbox"
+  root "emails#inbox"
 
-  get "done" => "pages#done", :as => :done
-  get "team" => "pages#team", :as => :team
-  get "calendar" => "pages#calendar", :as => :calendar
-  get "vip" => "pages#vip", :as => :vip
-  get "sent" => "pages#sent", :as => :sent
-  get "later" => "pages#later", :as => :later
-  get "draft" => "pages#draft", :as => :draft
-  get "archive" => "pages#archive", :as => :archive
-  get "trash" => "pages#trash", :as => :trash
+  get "all" => "emails#all", :as => :all
```

Comment:
Not a big deal, but this is old syntax! I know that for whatever reason the code you're removing is also the old syntax haha. Let's update this to the format: 

`get "all", to: "emails#all", as: :all`

Reply:
Done, updated the existing ones to the new syntax as well